recommenders/utils/python_utils.py

```diff
@@ -63,16 +75,82 @@ def lift(cooccurrence):
         numpy.ndarray: The matrix of Lifts between any two items.
     """
 
-    diag = cooccurrence.diagonal()
-    diag_rows = np.expand_dims(diag, axis=0)
-    diag_cols = np.expand_dims(diag, axis=1)
+    diag_rows, diag_cols = _get_row_and_column_matrix(cooccurrence.diagonal())
 
     with np.errstate(invalid="ignore", divide="ignore"):
         result = cooccurrence / (diag_rows * diag_cols)
 
     return np.array(result)
 
 
+def mutual_information(cooccurrence):
+    """Helper method to calculate the Mutual Information of a matrix of co-occurrences.
+
```

Comment:
Can we add here what is the effect of all these new metrics? Maybe explain the difference between them as we have in jaccard, lift, etc https://github.com/microsoft/recommenders/blob/staging/recommenders/utils/python_utils.py#L30

Let @anargyri or me know if you need support with the explanations

Reply:
Added